codice/ddf
# DDF-2294 Fix metatype reading single entry as multiple

#### What does this PR do?
Escapes the default metatype values. Previously it was reading each entry as 2 values, when it should be one.

#### Who is reviewing it? 
@ethantmanns @paouelle 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams

#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@coyotesqrl

#### How should this be tested?

Start up DDF and verify for the ValidationFilterPlugin and MetacardValidationPlugin metatypes have 1 entry instead of 2.

#### Any background context you want to provide?

#### What are the relevant tickets?
[DDF-2294](https://codice.atlassian.net/browse/DDF-2294)

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
